include/NovelRT/Utilities/Event.h

```diff
@@ -53,25 +58,37 @@ namespace NovelRT::Utilities {
       return _handlers.size();
     }
 
-    void operator+=(const EventHandler<TArgs...>& handler) {
+    Atom operator+=(const EventHandler<TArgs...>& handler) {
```

Comment:
I'm a bit concerned this may alienate some people - traditionally += doesn't return anything I don't think? so I'm not sure if this would make for good API UX. I'll wait and see on any sample code that may end up in the proposal though.

Reply:
I agree that's pretty weird, I'll put it back to void, however, there will still be no way of removing handlers created using the `operator+=` with `std::function`, which is something I would like to "fix", or we can maybe leave it as it is by design.